src/NzbDrone.Core/Download/TrackedDownloads/TrackedDownloadService.cs

```diff
@@ -57,23 +58,33 @@ public TrackedDownload TrackDownload(DownloadClientDefinition downloadClient, Do
             try
             {
                 var parsedEpisodeInfo = Parser.Parser.ParseTitle(trackedDownload.DownloadItem.Title);
-                if (parsedEpisodeInfo == null) return null;
+                var historyItems = _historyService.FindByDownloadId(downloadItem.DownloadId);
 
-                var remoteEpisode = _parsingService.Map(parsedEpisodeInfo);
+                if (parsedEpisodeInfo != null)
+                {
+                    trackedDownload.RemoteEpisode = _parsingService.Map(parsedEpisodeInfo);
+                }
 
-                if (remoteEpisode.Series == null || !remoteEpisode.Episodes.Any())
+                if (historyItems != null && !historyItems.Empty())
```

Comment:
also, `_historyService.FindByDownloadId` should never return null, if it does then it's a bug that should be fixed. our rule is to always return empty lists instead of null where the return type is enumerable.